### PR TITLE
Fix additional issues with the Bokeh tests

### PR DESCRIPTION
**Pull request by guynir42**
Addressing a few comments made after the previous PR was merged (#2640)

### CODE REVIEW

File: skyportal/tests/api/test_bokeh_plots.py [hunk 1]
```diff
@@ -31,21 +37,35 @@ def get_plot_data(data):
...
+    Parameters
+    ----------
+    data: dict
+       A dictionary loaded from the JSON file sent by the plotting API call.
+
+    Returns
+    -------
+    list
+        Returns a list of dictionaries with the flux, wavelength, etc.
+        Each dictionary for a different spectra.
```
Comment (mcoughlin) [inline skyportal/tests/api/test_bokeh_plots.py:49]: One dictionary per spectra?

File: skyportal/handlers/api/internal/plot.py [hunk 2]
```diff
@@ -47,10 +47,7 @@ def get(self, obj_id):
         device = self.get_query_argument("device", None)
         smoothing = self.get_query_argument("smoothing", False)
         if isinstance(smoothing, str):
-            if smoothing.lower() == 'true':
-                smoothing = True
-            else:
-                smoothing = False
+            smoothing = True if smoothing.lower() == "true" else False
```
Comment (mcoughlin) [inline skyportal/handlers/api/internal/plot.py:50]: looks like Stefan.

**Review by mcoughlin — APPROVED** [review]

File: skyportal/handlers/api/internal/plot.py (hunk 2 above)
Comment (guynir42) [reply skyportal/handlers/api/internal/plot.py:50]: Indeed.

File: skyportal/handlers/api/internal/plot.py [hunk 3]
```diff
@@ -47,10 +47,7 @@ def get(self, obj_id):
         device = self.get_query_argument("device", None)
         smoothing = self.get_query_argument("smoothing", False)
         if isinstance(smoothing, str):
```
Comment (stefanv) [inline skyportal/handlers/api/internal/plot.py:49]: remove the isinstance call, and set `smoothing = self.get_query_argument("smoothing", "false")`

File: skyportal/tests/api/test_bokeh_plots.py [hunk 4]
```diff
@@ -9,11 +9,17 @@ def black_body(la, temp):
     """
     get the amount of radiation expected from a black body
     of the given temperature "temp", at the given wavelengths "la".
-    :param la: float array or scalar
+
+    Parameters
+    ----------
+    la: float array or scalar
```
Comment (stefanv) [inline skyportal/tests/api/test_bokeh_plots.py:15]:
```suggestion
    la : float array or scalar
```

File: skyportal/tests/api/test_bokeh_plots.py [hunk 5]
```diff
@@ -9,11 +9,17 @@ def black_body(la, temp):
...
         wavelength(s) where the radiation should be calculated.
-    :param temp: float scalar
+    temp: float scalar
```
Comment (stefanv) [inline skyportal/tests/api/test_bokeh_plots.py:17]:
```suggestion
    temp : float scalar
```

File: skyportal/tests/api/test_bokeh_plots.py [hunk 6]
```diff
@@ -9,11 +9,17 @@ def black_body(la, temp):
...
         temperature of the black body.
-    :return:
+
+    Returns
+    -------
+        float array
```
Comment (stefanv) [inline skyportal/tests/api/test_bokeh_plots.py:22]:
```suggestion
array of float
```

Comment (stefanv) [reply skyportal/tests/api/test_bokeh_plots.py:22]: Fix indentation

File: skyportal/tests/api/test_bokeh_plots.py [hunk 7]
```diff
@@ -31,21 +37,35 @@ def get_plot_data(data):
     to api/internal/plot/spectroscopy/<obj_id> and recover
     the underlying data for all spectra in the plot.
 
-    Returns a list of dictionaries with the flux, wavelength, etc.
+    Parameters
+    ----------
+    data: dict
```
Comment (stefanv) [inline skyportal/tests/api/test_bokeh_plots.py:42]:
```suggestion
    data : dict
```

**Review by stefanv — COMMENTED** [review]
The enumerate is still there; can we also replace that now that the data is a numpy array?

**Comment by profjsb** [comment]
@guynir42 can you resolve merge conflicts here?

**Review by stefanv — APPROVED** [review]

**Pull request merged**